misk-aws-dynamodb-testing/src/main/kotlin/misk/aws/dynamodb/testing/InProcessDynamoDbModule.kt

```diff
@@ -1,9 +1,21 @@
 package misk.aws.dynamodb.testing
 
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBStreams
+import com.amazonaws.services.dynamodbv2.local.shared.access.AmazonDynamoDBLocal
+import com.amazonaws.services.dynamodbv2.local.shared.access.LocalDBClient
+import com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess
+import com.amazonaws.services.dynamodbv2.local.shared.jobs.JobsRegister
+import com.google.inject.Provides
+import java.io.File
+import java.util.concurrent.ExecutorService
+import javax.inject.Named
+import javax.inject.Singleton
+import kotlin.reflect.KClass
 import misk.ServiceModule
+import misk.concurrent.ExecutorServiceFactory
 import misk.inject.KAbstractModule
 import misk.inject.toKey
-import kotlin.reflect.KClass
 
 /**
  * Executes a DynamoDB service in-process per test. It clears the table content before each test
```

Comment:
Does this comment cover how it isolates side effects for concurrent test execution?

Reply:
oooh, good question. With this change we’re a bit more shielded from concurrent execution because we’re not binding ports or anything.